dali/test/python/test_utils.py

```diff
@@ -68,6 +69,24 @@ def get_device_memory_info(device_id=0):
         return None
 
 
+def get_gpu_name_from_nvml():
+    try:
+        nvml_output = subprocess.check_output(
+            ["nvidia-smi", "--query-gpu=name", "--format=csv,noheader"]
+        )
+        return nvml_output.decode("utf-8").strip().split("\n")
+    except subprocess.CalledProcessError:
+        return None
+
+
+def skip_if_m60():
+    """Skip the test if the GPU is M60. The video decoder is not supported in full on M60."""
+
+    gpus = get_gpu_name_from_nvml()
+    if "Tesla M60" in gpus:
+        raise SkipTest()
```

Comment:
Can't we just restrict the entire family of GPUs with compute capability <6.0? AFAIK Tesla M60 is a Maxwell, so it must be 5.x. Are there any pre-6.0 machines on which the tests did work?

Reply:
Meh. Approved, but I'm not comfortable with hardcoding things specific to our CI in our main source repo.